### DISCUSSION TITLE
Are Gantt charts possible?

### DISCUSSION THREAD

**Discussion by jccantu8** [jtablesaw/tablesaw#1038] [Q&A] [answered]
Hello, plotly natively supports Gantt charts through the [plotly.express interface](https://plotly.com/python/gantt/) - Are these possible in Tablesaw? I tried first creating a layout with an xAxis of type Date, but I wasn't able to figure out the bars themselves... Looking at [BarTrace in the api](https://www.javadoc.io/doc/tech.tablesaw/tablesaw-jsplot/0.36.0/tech/tablesaw/plotly/traces/BarTrace.BarBuilder.html) didn't seem to suggest that this is possible. Can anyone confirm for sure? Thanks.

**Comment by lwhite1** [comment, marked as answer]
I don’t believe it is possible currently.  Plotless has some higher level
APIs that are used to implement complex plots.  These have t been ported.

**Reply by jccantu8** [reply]
Thanks, @lwhite1, I figured that was the case.
